### PR TITLE
[FUSED] Add reciprocal op

### PR DESCRIPTION
**Pull request by tsocha**

### CODE REVIEW

File: src/ngraph/op/fused/reciprocal.hpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            {
+            public:
+                NGRAPH_API
+                static constexpr NodeTypeInfo type_info{"Reciprocal", 1};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                Reciprocal() = default;
+                /// \brief Constructs a Reciprocal operation.
+                ///
+                /// \param data Input tensor
+                /// \param slope Multipliers for negative values
```
[the remark this answers is not in the document]
Comment (tsocha) [reply src/ngraph/op/fused/reciprocal.hpp]: Done.

**Comment by tsocha** [comment]
> @diyessi 
> Remove the "slope" param description.

Done.

File: src/ngraph/op/fused/reciprocal.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    : FusedOp({data})
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::Reciprocal::decompose_op() const
+{
+    auto data = input_value(0);
+    auto one_node = op::Constant::create(data.get_element_type(), data.get_shape(), {1});
+    return {make_shared<op::Divide>(one_node, data, AutoBroadcastType::NUMPY)};
```
Comment (GlebKazantaev) [inline src/ngraph/op/fused/reciprocal.cpp]: Let's use V1::Divide here

Comment (tsocha) [reply src/ngraph/op/fused/reciprocal.cpp]: Done

**Pull request merged**